andela/colossus-backend
# #167733806 Integrate coveralls & add code coverage badge to Readme

**What does this PR do?**
integrates coveralls coverage and adds coverage badge to Readme

**Description of task to be completed**
Turn on the colossus-backend repo in my coveralls account
copy the badge markdown and add to Readme
Install the related npm packages coveralls, nyc, mocha-lcov-reporter

**How should this be manually tested**
check the Readme to ensure the badge is visible
coverage should be visible if all went well

**What are the relevant Pivotal tracker stories**
#167733806
